server-embedded/src/main/java/org/exoplatform/chat/server/ChatServer.java

```diff
@@ -113,7 +115,23 @@ public ChatServer()
   {
     return index.ok();
   }
-
+  
+  @Resource
+  @Route("/filterOutSilentUsers")
+  public Response.Content getFilteredList(String user,List<String> roomUsers, String roomId, String token){
+	    if (!tokenService.hasUserWithToken(user, token)){
+	      return Response.notFound("Petit malin !");
+	    }
+	  List<String> receivers =  new ArrayList<>();
+	  for(String participant : roomUsers.get(0).split(",")) {
+		  if(!notificationService.isRoomSilentForUser(participant, roomId))
+			  receivers.add(participant);
+	  }
+	  return Response.ok(new Gson().toJson(receivers)).withMimeType(MIME_TYPE_JSON ).withHeader
```

Comment:
i used the google.gson library to convert the list of strings 'receivers' to a json object

Reply:
Can you try without adding external library ?
1st solution : 
```suggestion
	  return Response.ok(receivers).withMimeType(MIME_TYPE_JSON ).withHeader
```
Not sure if this is working

2nd possibility : use JSONObject class which already exists in the class

Reply:
passing the list as is in the response doesn't work, we have to convert it to json first

Reply:
so please use jsonObject instead of new library

Reply:
i've used the JSONArray class which is inside an already imported library and it has the toJsonString method that does exactly what weneed.
check the last commit